third_party/packages/flutter_svg/pubspec.yaml

```diff
@@ -13,9 +13,9 @@ dependencies:
     sdk: flutter
   http: ^1.0.0
   # See https://github.com/flutter/flutter/issues/157626
-  vector_graphics: ">=1.1.11+1 <= 1.1.12"
+  vector_graphics: ">=1.1.11+1 <= 1.1.14"
```

Comment:
Same notes here as in the other package.

Reply:
the same as https://github.com/flutter/packages/pull/8014#discussion_r1827783576